src/bridge/cockpitdbuscache.c

```diff
@@ -1040,6 +1041,12 @@ on_properties_signal (GDBusConnection *connection,
   g_debug ("%s: signal PropertiesChanged at %s", self->logname, path);
   g_variant_get (body, "(&s@a{sv}@as)", &interface, NULL, NULL);
 
+  /* We have to filter the sender ourselves; glib doesn't do that for
+     well-known names.
+   */
+  if (self->name_owner && g_strcmp0 (sender, self->name_owner) != 0)
```

Comment:
FTR, the whole point of strcmp0 is that you don't have to do the NULL check explicitly. (Harmless of course, and hopefully the compiler just optimizes it away)

Reply:
It's necessary here since we want to skip the return when name_owner is NULL (very unlikely in practice, but possible).

Reply:
Of course, totally mis-read that :blush